src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/PrisonOffenderManager.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.delius.jpa.standard.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+@EqualsAndHashCode(of = "prisonOffenderManagerId")
+@ToString(exclude = {"team","staff","probationArea", "responsibleOfficer" ,"managedOffender"})
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@Entity
+@Table(name = "PRISON_OFFENDER_MANAGER")
+public class PrisonOffenderManager {
+
+    @Id
+    @Column(name = "PRISON_OFFENDER_MANAGER_ID")
+    private Long prisonOffenderManagerId;
+
+    @JoinColumn(name = "ALLOCATION_TEAM_ID")
+    @OneToOne
+    private Team team;
```

Comment:
Double check this shouldn't point to AllTeam - I am only just getting my head round the differences between Team and AllTeam and I'm getting the impression AllTeam should be used but I think we need to do a bit more digging..

Reply:
Good point. I was confused by the AllTeam relationships here. I am thinking that it might be better to send this out (low risk) and let HDC check whether the data they see matches up with the data they currently get from MI extracts. I can then investigate any differences and correct the relationships if required?